localdev/test.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+rm -rf ../sandbox/src/*
+rm -rf ../sandbox/target/*
+
+docker compose down #incase
+
+mkdir ../sandbox/src/existBeforeStart
+echo "some content" > ../sandbox/src/existBeforeStart/some.txt
+
+docker compose up --build -d --wait
+sleep 2 # so the container has time do initial copying and initialize the watches
+
+mkdir ../sandbox/target/music
+
+success="true"
+cd ../sandbox/src
```

Comment:
do not rely on sandbox, it is for personal use, create a directory called volumes in this directory

you also do not need to create directories, docker will create them if missing when you use bind mounts